app/assets/javascripts/Components/Result/marks_panel.jsx

```diff
@@ -259,7 +259,7 @@ class FlexibleCriterionInput extends React.Component {
              </span>;
     });
 
-    if (this.props['marks.override']) {
```

Comment:
Now that you've simplified the property names, you can use dot notation to access them: `this.props.override`, etc. (This is true throughout)